plugin-server/tests/main/ingestion-queues/analytics-events-ingestion-consumer.test.ts

```diff
@@ -109,23 +114,23 @@ describe('eachBatchParallelIngestion with overflow reroute', () => {
 
     it('reroutes excess events to OVERFLOW topic', async () => {
```

Comment:
Can't quite test this by cherry-picking, but https://github.com/PostHog/posthog/pull/20945#discussion_r1548779396 would have caused this test to hang indefinitely now that it actually produces -- this would have let us know something weird was going on earlier.